src/main/java/blackjack/domain/Card.java

```diff
@@ -0,0 +1,27 @@
+package blackjack.domain;
+
+public class Card {
+    private final CardNumber cardNumber;
+    private final CardSymbol cardSymbol;
+
+    public Card(CardNumber cardNumber, CardSymbol cardSymbol) {
+        this.cardNumber = cardNumber;
+        this.cardSymbol = cardSymbol;
+    }
+
+    public boolean isACE() {
+        return cardNumber.number().equals(CardNumber.ACE.number());
+    }
```

Comment:
```suggestion
    public boolean isAce() {
        return cardNumber == CardNumber.ACE;
    }
```
enum의 값을 비교하기 보다는 enum 자체로 비교할 수 있습니다

Reply:
오 이런식으로 쉽게 값을 비교할 수 있군요 .. ! 참고하여 수정하였습니다 !